app/models/manageiq/providers/azure/cloud_manager/provision/state_machine.rb

```diff
@@ -0,0 +1,9 @@
+module ManageIQ::Providers::Azure::CloudManager::Provision::StateMachine
+  def customize_destination
```

Comment:
So, this method gets called after the VM is created right?

I wonder how [this method call](https://github.com/durandom/manageiq/blob/88bb8949a05b99915b03e37e4f0a6f82404db6ae/app/models/manageiq/providers/cloud_manager/provision/state_machine.rb#L67-L67) does not break the other providers, because I only see it defined in openstack and amazon?